collections/Quests/NumberInput/index.ts

```diff
@@ -0,0 +1,41 @@
+import { CalculatorIcon } from '@heroicons/react/outline'
+import { IQuest } from '@/collections/types'
+import EditView from './EditView'
+import PlayView from './PlayView'
+import { useQuestStore } from './store'
+
+export type NumberInputQuestData = {
+  question: string
+  answer: number
+  unit: string
+}
+
+const NumberInputQuest: IQuest<NumberInputQuestData> = {
+  type: 'QUEST_NUMBER_INPUT',
```

Comment:
```suggestion
  type: 'QUEST_STATISTICS',
```
Lieber vordefinierte Typen verwenden, sonst funktioniert das Ausführen der Quests nicht.

Reply:
Die QUEST_NUMBER_INPUT ist im schema.prisma definiert. Mit einem `prisma generate` wird dein locales prisma schema geupted mit der value